aspnetcore/blazor/javascript-interop.md

```diff
@@ -1,26 +1,26 @@
 ---
-title: Razor Components JavaScript interop
+title: Blazor JavaScript interop
```

Comment:
For file rename
.../_static/aspnet-core-razor-components.png → .../_static/aspnet-core-razor-components.png

why leave RC in the file name?

Reply:
Because *today*, this is the case. They don't see the file name, but I agree *soon* that this should be updated. The whole direction is going to go "client-side" and "server-side" shortly. *Today* tho, we're in a bit of *limbo* on these concepts. Next PR should take care of this.